crawler.js

```diff
@@ -43,6 +44,7 @@ class Crawler {
 
     this.userAgent = "";
     this.behaviorsLogDebug = false;
+    this.profileDir = fs.mkdtempSync(path.join(os.tmpdir(), "profile-"));
```

Comment:
so will the profile information be stored in this temp folder in the docker image?

Reply:
yeah, the profile is created in a temp directory, but then exported as a tarball and copied to the location specified by `--filename`. Need to add docs to explain this, and ideally, add a test as well..